packages/eslint-config/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "@talend/eslint-config",
+  "version": "0.1.0",
```

Comment:
Semver has deprecated versions 0.X.Y, versions should start with at least 1 :)
More on this here https://github.com/semver/semver/issues/221
(I don't give a damn, especially for a tooling project but it's interesting nonetheless)

Reply:
Ok let's set 1.0.0